templates/confidential-computing/index.html

```diff
@@ -295,41 +315,47 @@ <h2>Use Ubuntu Pro to further harden your confidential VMs</h2>
           <p>
             While confidential VMs safeguard your workload from external threats, internal vulnerabilities within their boundaries can still pose risks. This is where Ubuntu Pro proves invaluable, ensuring your guest CVM software stack is continuously patched and up-to-date.
           </p>
+          <hr class="p-rule--muted" />
           <p>
             <a href="/pro">Learn more about Ubuntu Pro&nbsp;&rsaquo;</a>
           </p>
         </div>
       </div>
     </section>
 
-    <hr class="is-fixed-width" />
-
     <section class="p-strip u-no-padding--top">
-      <div class="row--50-50">
+      <hr class="p-rule is-fixed-width" />
+      <div class="row--50-50 u-sv3">
         <div class="col">
-          <h2>A solid foundation of your zero trust strategy</h2>
+          <h2>A solid foundation of your<br/> zero trust strategy</h2>
         </div>
         <div class="col">
           <p>
-            With confidential computing, you can remove the privileged systems
-            software from your trusted computing base, reduce your attack surface,
-            and get a remotely verifiable cryptographic guarantee before trusting
-            any platform with your data.
+            With confidential computing, you can remove the privileged systems software from your trusted computing base, reduce your attack surface, and get a remotely verifiable cryptographic guarantee before trusting any platform with your data.
           </p>
+          <hr class="p-rule--muted" />
           <p>
-            <a href="/blog/build-foundation-zero-trust-strategy-ubuntu-confidential-computing">Learn
-            more about zero trust and confidential computing&nbsp;&rsaquo;</a>
+            <a href="/blog/build-foundation-zero-trust-strategy-ubuntu-confidential-computing">Learn more about zero trust and confidential computing&nbsp;&rsaquo;</a>
           </p>
         </div>
       </div>
+      <div class="row">
+        {{ image(url="https://assets.ubuntu.com/v1/e3769453-server-room.png",
+          alt="Server room with racks of servers",
```

Comment:
Same here, not needed as it's decorative
```suggestion
          alt="",
```